pkgs/tools/system/hw-probe/default.nix

```
@@ -0,0 +1,115 @@
+{ config
+, stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+
+# Required
+, hwinfo
+, dmidecode
+, smartmontools
+, pciutils
+, usbutils
+, edid-decode
+
+# Conditionally recommended
+, systemdSupport ? config.systemdSupport or stdenv.hostPlatform.isLinux
```

If this is agreeable to other contributors I'll follow up with a PR to add `systemdSupport ? config.systemdSupport or stdenv.hostPlatform.isLinux` to other packages that have this flag.

Following the [advice in the manual](https://nixos.org/manual/nixpkgs/stable/#ssec-cross-platform-parameters) I've opted to go with `stdenv.hostPlatform.isLinux` instead of `stdenv.isLinux`.

---

> If this is agreeable to other contributors I'll follow up with a PR to add `systemdSupport ? config.systemdSupport or stdenv.hostPlatform.isLinux` to other packages that have this flag.

We should solve that with a RFC first because right now there is no real standard.

```suggestion
, systemdSupport ? stdenv.isLinux
```